libs/teams/teams-chat-workflow-spring-boot-starter/src/main/java/org/finos/springbot/teams/handlers/TeamsResponseHandler.java

```diff
@@ -62,7 +61,7 @@ public class TeamsResponseHandler implements ResponseHandler, ApplicationContext
 	protected ThymeleafTemplateProvider displayTemplater;
 	protected TeamsStateStorage teamsState;
 	protected TeamsConversations teamsConversations;
-	protected MessageRetryHandler messageRetryHandler;
+	protected MessageRetryHandler retryHandler;
```

Comment:
can we not have retryHandler in a subclass?  e.g. `RetryingTeamsResponseHandler`?

Reply:
Changes done as per review comments